### PR TITLE
WIP Add tutorial about normal gravity and free-air correction

### PR DESCRIPTION
**Pull request by santisoler**
Add tutorial on the use of the closed-form formula for normal gravity and a comparison with the free-air correction (first-order approximation).
Modify doc configuration and index files to include the new tutorial pages.

**Reminders**:

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst` and the base `__init__.py` file for the package.
- [ ] Write detailed docstrings for all functions/classes/methods. It often helps to design better code if you write the docstrings first.
- [ ] If adding new functionality, add an example to the docstring, gallery, and/or tutorials.
- [ ] Add your full name, affiliation, and [ORCID](https://orcid.org) (optional) to the `AUTHORS.md` file (if you haven't already) in case you'd like to be listed as an author on the [Zenodo](https://zenodo.org/communities/fatiando) archive of the next release.

### CODE REVIEW

**Comment by santisoler** [comment]
This could be inside FAQs section instead of a tutorial. We should add a tutorial on how to compute gravity disturbance and link to this FAQ at the end with a side note on why not use free-air correction.

**Comment by LL-Geo** [comment]
I'm thinking to put the tutorial of gravity disturbance and free-air gravity like this. https://github.com/LL-Geo/PFToolbox/blob/master/Gravity%20disturbance.ipynb
Keen for comments and suggestions 😄 @leouieda @santisoler

**Comment by santisoler** [comment]
@LL-Geo I definitely think that the tutorial about normal gravity should be something like the Notebook you wrote. While the explanation about free-air vs analytical solution for normal gravity should be something like a FAQ. So feel free to use anything from this PR, I'll probably close it on behalf of other PR that do this job correctly.

File: tutorials/free_air_correction.py [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+# observed gravity vector and the norm of the normal gravity vector on the same
+# observation point :math:`\mathbf{p}` (Bathelmes, 2013):
+#
+# .. math::
+#
+#   \delta g(\mathbf{p})
+#     = |\bar{g}_\text{obs}(\mathbf{p})| - |\bar{\gamma}(\mathbf{p})|
+#     = {g}_\text{obs}(\mathbf{p}) - \gamma(\mathbf{p})
+#
+# In order to compute the magntide of the normal gravity vector we should start
```
Comment (aguspesce) [inline tutorials/free_air_correction.py:28]:
```suggestion
# In order to compute the magnitude of the normal gravity vector we should start
```

File: tutorials/free-air.ipynb [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Normal gravity\n",
+    "\n",
+    "Gravity disturbance is defined as the difference between the norm of the observed gravity vector and the norm of the normal gravity vector on the same observation point $\\mathbf{p}$ (Bathelmes, 2013):\n",
+    "\n",
+    "$$ \\delta g(\\mathbf{p}) = |\\bar{g}_\\text{obs}(\\mathbf{p})| - |\\bar{\\gamma}(\\mathbf{p})| = {g}_\\text{obs}(\\mathbf{p}) - \\gamma(\\mathbf{p}) $$\n",
+    "\n",
+    "In order to compute the magntide of the normal gravity vector we should start by defining what do we understand as _normal Earth_.\n",
```
Comment (aguspesce) [inline tutorials/free-air.ipynb:40]: It has a typo in magnitude

**Comment by santisoler** [comment]
I'm closing this PR. It's been too long since we started it and it's probably harder to update the branch that to start it from scratch again. I'll leave the branch in the repo in case we want to retake this in the future.

**Pull request closed without merging**